Dockerfile

```diff
@@ -28,8 +35,10 @@ COPY --from=builder /build/v3 /var/www/html/v3
 
 # Offline build: cache .theia and .vsix files in registry itself and update metas
 FROM builder AS offline-builder
-RUN ./cache_artifacts.sh v3
-RUN chmod -R g+rwX /build
+
+# To only cache files from /latest/ folders, use ./cache_artifacts.sh v3 --latest-only 
+# and uncomment line above to remove files so they're not included in index.json -- RUN rm -fr $(find /build/v3 -name 'meta.yaml' | grep -v "/latest/" | grep -o ".*/")
```

Comment:
For the non-rhel dockerfile, this could be implemented as an ARG (`ONLY_LATEST=true`) to avoid having to edit the dockerfile for specific builds.